{{cookiecutter.project_slug}}/Makefile

```diff
@@ -1,8 +1,9 @@
 SHELL := /bin/bash
 
-PY_MODULE := {{ cookiecutter.module_slug }}
+PY_SRC_DIR := {{ cookiecutter.__project_src_path }}
+PY_IMPORT := {{ cookiecutter.__project_import }}
```

Comment:
Making sure I understand: `PY_SRC_DIR` is now the top-level source directory, and `PY_IMPORT` is the `import`-able name, right?

Ideally we'd collapse these into one state for the various Makefile targets (`PY_TOP`?), if the tools that need one or the other could use just one. But if not, this seems fine to me 🙂

Reply:
`PY_SRC_DIR` points to the directory directly containing the module's source files. This only really means something for namespace packages. So, in one of the CI examples/tests, where the namespace is `tob.r_and_e`, `PY_SRC_DIR` will be `tob/r_and_e/python_project`.

While developing this PR, I noticed that `mypy`, `pdoc`, and `pytest-coverage` seemed to play better with module/`import`-able names instead of paths (`PY_IMPORT`).

Regarding the comment about collapsing these two variables into one state, I'm not sure exactly what you're asking/expecting. Do you mean derive these two different strings from one variable, i.e. module/`import`-able contains `.` whereas source directory contains `/`?

Also, am I missing a use-case for allowing the user to override these? If we use `=` instead of `:=`, then they would not go out of sync.